src/TorchSharp/Tensor/TMath.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Do we actually need this class?

Reply:
Yes, wanted to ask about this one! In libtorch we can do things like:
return 0.5 * (1 + torch::exp(value / sqrt(2))); // libtorch
Was wondering if we need static Math-like class to do analogous thing in TorchSharp, e.g. 
return 0.5 * (1 + TMath.Exp(value / Math.Sqrt(2))); // TorchSharp?
Can add to TorchTenor, but perhaps becomes a little verbose and in C# we might expect a class with a name like 'Math' to be used? Please let me know what you prefer...

Reply:
Ah that a good point. I believe that what we can do is detaching the current C# API into a functional (basically containing the static methods calling into C) and an object oriented API (the current C# API). In this way one can basically pick which API to use. Some of the methods of course will only live in one of the API (like exp only on the functional one). WDYT?

Reply:
The lower level, all `static` function API would also be useful for a more idiomatic F# API, right?

Reply:
Exactly.

Reply:
#110